CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ## Enhancements
 
-* All Realm annotations are now kept at runtime, allowing runtime tools to access them (#5344).
+* All Realm annotations are now kept at runtime, allowing runtime tools access to them (#5344).
```

Comment:
previous wording was better, current wording is actually valid if it's `allowing runtime tools' access to them` :open_mouth: 

Enforcing grammar is so strange though!